apps/web/src/features/applications/components/application-ai-copilot.tsx

```
@@ -20,9 +18,9 @@ import { applicationsListQueryKey } from "../queries";
 
 // Score bands drive the ring color and the label — a job-fit gauge, not a generic percentage.
 function band(score: number) {
-	if (score >= 75) return { color: "#10b981", label: t`Strong fit` };
-	if (score >= 50) return { color: "#f59e0b", label: t`Worth a shot` };
-	return { color: "#f43f5e", label: t`A stretch` };
+	if (score >= 75) return { color: "#10b981", label: `Strong fit` };
```

Biome (2.5.6)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)